WolvenKit.Common/Tools/FileHelper.cs

```diff
@@ -151,4 +151,45 @@ public static string SanitizePath(string path)
         inputPath = string.Join(Path.DirectorySeparatorChar, inputPathArray);
         return inputPath;
     }
+
+
+    public static string? GetFirstExistingParentDirectory(string path)
```

Comment:
Maybe I'm blind but where is this function used?

Reply:
It's not, you're right, this can go